### DISCUSSION TITLE
Pass through kitty graphics protocol in preview pane

### DISCUSSION THREAD

**Discussion by saulrh** [PatrickF1/fzf.fish#283] [Ideas]
I use kidonng's [preview.fish](https://github.com/kidonng/preview.fish) to preview file contents in the directory view. It uses `timg` to preview anything with an `image/` mime type, and it works fairly well overall:

![image](https://user-images.githubusercontent.com/560724/207994685-5eef4994-c091-485f-acd4-60dfebd03fc0.png)

However, it could work _better_, because I'm running kitty and have its [terminal graphics protocol](https://sw.kovidgoyal.net/kitty/graphics-protocol/). If you're running iterm2 you have something comparable. If I hack `preview` to force `timg` to output the kitty terminal graphics protocol (`-pk`), it looks like the control sequences get stripped out or otherwise lost in between `timg` and the preview pane and I get junk:

![image](https://user-images.githubusercontent.com/560724/207994545-8a3557da-e424-4739-9f2f-5face3dd06ad.png)

It'd be _really nifty_ if this worked:

![image](https://user-images.githubusercontent.com/560724/207994907-f47f7854-e582-4a16-ac70-9df607491367.png)

**Comment by PatrickF1** [comment]
I think with [fzf 0.44](https://github.com/junegunn/fzf/releases/tag/0.44.0) this might be supported now. But anyway if it's not it's probably a fzf limitation and not fzf.fish. Good luck!
